test/storage/garbage_collector_test.cpp

```diff
@@ -1176,14 +1190,14 @@ TEST_F(GarbageCollectorTests, DeleteTest1) {
   }
 }
 
+/*
+ * Consider performing GC on the version chain sorted from newest to oldest:
+ * DELETE
+ * txn1 <- active txn
```

Comment:
Consistency with the previous notation
```suggestion
 T1 <- active txn
```